doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -745,16 +749,26 @@ the right end is a term shall use by default rules at level strictly
 less than ``n`` to parse this sub-term if the rule is declared left
 associative and rules at level less or equal than ``n`` if the rule is
 declared right associative. This is what happens for instance in the
-rule ``Notation "x + y" := (Add x y) (in custom expr at level 2, left
-associativity)`` where ``x`` is any expression parsed in entry
+rule
+
+.. coqtop:: in
+
+   Notation "x + y" := (Add x y) (in custom expr at level 2, left associativity).
```

Comment:
Reads better like this because this verbatim was too long and was breaking the paragraph flow. Same for the other similar changes.